src/components/ColorButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+
+const ColorButton = (props) => {
+  const color = props.color;
+  const icon = props.icon;
+  const updateColor = props.updateColor;
+
+  const changeColor = () => {
+    console.log('handleCliked on colorButton called');
+    updateColor();
+  };
```

Comment:
Because `updateColor` is already a function that takes no arguments we don't need to wrap it in another function. We can pass it directly to `onClick`.